lib/user_repository.rb

```diff
@@ -19,14 +20,32 @@ def list_handles
     return handles
   end
   
-  def updated_verified(user)
-    sql = 'UPDATE users SET verified = $1 WHERE id = $2;'
-    sql_params = [user.verified, user.id]
-    DatabaseConnection.exec_params(sql, sql_params) 
-  end
+  # def updated_verified(user)
```

Comment:
Can commented code be removed or is it needed later?

Reply:
This is probably not needed but something almost identical needed when we implement the automatic replies so worth leaving rather than writing from scratch later